plaid_checks.xml

```diff
@@ -91,7 +91,7 @@
           value="Type name ''{0}'' must match pattern ''{1}''."/>
     </module>
     <module name="MemberName">
-      <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9]*$"/>
+      <property name="format" value="^[a-z][a-zA-Z0-9]*$"/>
```

Comment:
the old regex would not allow single letter class member names which is required for webhook keys as there are keys named `x` and `y` in the JWK response

Reply:
ah i see